compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1967,6 +1968,31 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
                 ));
                 debug!("lower_async_fn_ret_ty: generic_params={:#?}", generic_params);
 
+                let is_async_send = this.tcx.features().async_fn_in_trait
+                    && (true
```

Comment:
is this needed?

Reply:
The `true` definitely isn't! (Sorry, that was left over from by debugging attempts).

The async_fn_in_trait check is probably not needed either since you can't even type this attribute without it. I'll try and remove it.